src/main/java/racingcar/race/RaceInformation.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.race;
+
+import racingcar.Car;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RaceInformation {
+    private int numberOfAttempt;
+    private List<Car> carInformations = new ArrayList<>();
+    private static RaceInformation raceInformation= new RaceInformation();
+
+    private RaceInformation() { }
+
+    public static RaceInformation getInstance() {
+        return raceInformation;
+    }
+    public void initRegisterInformation(int numberOfAttempt) {
+        this.numberOfAttempt = numberOfAttempt;
+    }
+
+    public int getNumberOfAttempt() {
+        return numberOfAttempt;
+    }
+
+    public void initCarInformations(String[] carNames) {
+        for (String carName : carNames) {
+            carInformations.add(new Car(carName));
+        }
+    }
+
+    public String[] splitCarNameBuffer(String carNameBuffer) {
+        return carNameBuffer.split(",");
```

Comment:
만약 콤마를 상수로 뺀다면 (상수명은 일단 대충 아래처럼 지었다고 치면) 아래와 같이 코드의 의미가 더 명확하겠지?
```suggestion
        return carNameBuffer.split(CAR_NAME_SPLITER);
```